application/src/main/java/org/opentripplanner/apis/transmodel/TransmodelGraphQLSchema.java

```diff
@@ -661,7 +660,12 @@ private GraphQLSchema create() {
               .build()
           )
           .argument(
-            GraphQLArgument.newArgument().name("authority").type(Scalars.GraphQLString).build()
+            GraphQLArgument
+              .newArgument()
+              .name("authority")
+              .description("The ID of the authority to fetch stops for.")
```

Comment:
```suggestion
              .deprecated("This is the Transmodel namespace or the GTFS feedID - avoid using this. Request a new field in necessary.")
```

Reply:
Making this change along with others locally.